mkdocs.yml

```diff
@@ -1,4 +1,4 @@
-site_name: Sequelize - node.js ORM for Postgres, MySQl, MariaDB and SQLite
+site_name: Sequelize | The Node.js ORM
```

Comment:
@sdepold ORM alone is too generic, we are SQL focused, that matters a great deal. Hence why it also matters that people might search for `Node postgres orm`